composer.json

```diff
@@ -26,10 +26,10 @@
     "homepage": "https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle",
     "require": {
         "php": "^7.4 || ^8.0",
-        "doctrine/dbal": "^2.13 || ^3.0",
+        "doctrine/dbal": "^3.4",
         "doctrine/doctrine-bundle": "^2.4",
```

Comment:
could also be bumped to `^2.8` maybe? 2.4 is not maintained anymore.

Reply:
2.8 only support sf 54 and 60.

2.7 is the maximum I can update this one. Changed.